{"uuid": "0f7840d2-6ff8-48d4-893b-13b44970dfe9", "children": ["4b86a576-d81a-40a6-912d-4a02b9bd6ae5"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "TypeError: WebDriver.__init__() got an unexpected keyword argument 'executable_path'\n", "trace": "  File \"/home/test/Desktop/sprint_7/venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/test/Desktop/sprint_7/venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/test/Desktop/sprint_7/venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/test/Desktop/sprint_7/conftest.py\", line 22, in driver\n    driver = webdriver.Firefox(executable_path=GeckoDriverManager().install())\n"}, "start": 1728419185933, "stop": 1728419188434}], "start": 1728419185933, "stop": 1728419188560}